stdlib/2/os/__init__.pyi

```diff
@@ -197,18 +197,20 @@ def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
                                                     List[AnyStr]]]: ...
 
 def abort() -> None: ...
-def execl(file: _PathType, *args) -> None: ...
-def execle(file: _PathType, *args) -> None: ...
-def execlp(file: _PathType, *args) -> None: ...
-def execlpe(file: _PathType, *args) -> None: ...
+def execl(path: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> NoReturn: ...
+def execle(path: _PathType, arg0: Union[bytes, Text],
+           *args: Any) -> NoReturn: ...  # Imprecise signature
```

Comment:
The "imprecision" here is apparently that the last argument is treated as the env and must be a mapping, but we can't express that kind of signature. Keeping it as Any is fine, but we need to remove arg0 here too, since giving just the path and the env is valid.